eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionWriterTest.java

```diff
@@ -61,9 +63,10 @@ public void setUp() throws Exception {
     }
 
     @Test
+    @DirtiesContext
     public void saveSingleAccession() throws Exception {
         SubmittedVariant variant = new SubmittedVariant("accession", TAXONOMY, "project", "contig", 100, "reference",
```

Comment:
If the taxonomy is extracted into a constant to make the parameters more readable, I think the position should also be. In fact they are implemented that way in `VariantProcessorTest`. Applies to all the tests in this class.

Reply:
that is already done in another branch